src/components/DealDetailsStep.tsx

```diff
@@ -292,7 +303,17 @@ export const DealDetailsStep: FunctionComponent<DealDetailsStepProps> = ({
 				className="mt-8"
 				icon={<Icon name="plus-square" size={IconDimension.MIDDLE} />}
 				onClick={() =>
-					onNextStep(["dealName", "borrower", "principal", "financingFee", "timeToMaturity"], 1)
+					onNextStep(
+						[
+							"dealName",
```

Comment:
I would prefer it if we made a map/object with these strings as values so if we need to change them it is applied everywhere.